vetiver/templates/model_card.qmd

```diff
@@ -0,0 +1,153 @@
+---
+title: "Model Card: your model title"
+output:
+  html_document
+---
+```{python}
+#| echo: false
+from IPython.display import display, Markdown
+import datetime
+display(Markdown(f"""
+###### {datetime.date.today()}
+"""))
+```
+
+```{python}
+#| tags: [parameters]
+#| include: false
+import pins
+
+raw = "https://colorado.rstudio.com/rsc/"
+paths = {"v": "sacramento/"}
+board = pins.board_urls(raw, paths, allow_pickle_read=True)
+```
+
+```{python}
+#| include: false
+import datetime
+import pandas as pd
+import plotly.express as px
+from sklearn import metrics
+import vetiver
+
+v = vetiver.VetiverModel.from_pin(board, "v")
+v_meta = board.pin_meta("v")
+```
+
+
+A [model card](https://doi.org/10.1145/3287560.3287596) provides brief, transparent, responsible reporting for a trained machine learning model.
+
+## Model details
+
+- Developed by PERSON AND/OR TEAM
+```{python}
+#| echo: false
+display(Markdown(f"""
+- A {v.description} using {len(v.ptype.construct().dict())} features.
+"""))
+```
```

Comment:
Do you think it would be nicer to generate any of this in that setup chunk (getting `v` and `v_meta`, etc)? Or better to have it really close to the text like this? I'm noticing that this is more to read/parse than `` `r v$description` `` so was thinking what might be the best user experience for the Python Quarto person.

Also is there a way to pluralize this? For when there is one feature? (Common in text models, for example)

Reply:
I think it's too far away in the setup chunk, but pulled them out in the chunk of use. 👍 much more readable now!